src/components/calendar/EventDetailModal.tsx

```
@@ -1084,6 +1097,7 @@ export function EventDetailModal({
                           '[&_blockquote]:border-l-4 [&_blockquote]:border-muted [&_blockquote]:pl-4 [&_blockquote]:italic [&_blockquote]:my-2',
                           '[&_hr]:my-4 [&_hr]:border-t [&_hr]:border-border'
                         )}
+                        style={{ wordBreak: 'break-all', overflowWrap: 'anywhere' }}
                         dangerouslySetInnerHTML={{ __html: desc }}
```

Biome (2.3.14)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)